src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionProviderOptionsHelper.cs

```diff
@@ -48,8 +48,9 @@ internal static ConnectionProviderOptions BuildConnectionProviderOptions()
                         ValueType = ConnectionOption.ValueTypeCategory,
                         SpecialValueType = ConnectionOption.SpecialValueAuthType,
                         CategoryValues = new CategoryValue[]
-                        { new CategoryValue {DisplayName = "SQL Login", Name = "SqlLogin" },
-                          new CategoryValue {DisplayName =  "Windows Authentication", Name= "Integrated" }
+                        { new CategoryValue { DisplayName = "SQL Login", Name = "SqlLogin" },
+                          new CategoryValue { DisplayName = "Windows Authentication", Name = "Integrated" },
+                          new CategoryValue { DisplayName = "Azure Active Directory (MFA)", Name = "AzureMFA" }
```

Comment:
Should we use the same text as SSMS?  I think this corresponds to the `Active Directory - Universal with MFA support`.

Reply:
In our design meeting we had agreed it should include the word "Azure" at the start but I can update it to be "Azure Active Directory - Universal with MFA support"

Reply:
+1